DEVELOPMENT.md

```diff
@@ -1,8 +1,3 @@
-**NOTE: Do NOT expect the CI machine to build without making some manual changes
```

Comment:
I thought I'd removed this with the last commit; this isn't needed anymore as we have, in fact, rebuilt a CI machine (and just to make sure, I rebuilt a test CI machine as well using the configuration as it exists now).